docs/sources/features/datasources/cloudwatch.md

```diff
@@ -162,6 +168,29 @@ Left clicking a time series in the panel shows a context menu with a link to `Vi
 
 This feature is not available for metrics that are based on metric math expressions.
 
+## Using the Logs Query Editor
+
+> Only available in Grafana v7.0+.
+
+To query CloudWatch logs you have select region and up to 20 log streams which you want to query. Use the main input area to write your query in [CloudWatch Logs Query Language](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_QuerySyntax.html)
+
+You can also write queries returning time series data by using [`stats` command](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_Insights-Visualizing-Log-Data.html). When using `stats` queries in Explore, you have to make sure you are in Logs Explore mode.
+
+TODO: image Explore switcher
+
+To the right of the query input field is CloudWatch Insights link that will open CloudWatch Insights console with your query so you can continue exploration there if needed.
+
+TODO: image highlight link
+
+### Using template variable
+TODO:
```

Comment:
@kaydelaney Can you check this one? I assume this works but not exactly sure how. can you just reference the variables with the ${__var} syntax in the editor?

Reply:
Sure. I'll double check but yes I think you can use ${__var} syntax from within the editor.